content/articles/using-django-in-kivy-application/index.md

```diff
@@ -0,0 +1,596 @@
+### Using Django in Kivy applications.
+
+### Introduction.
+
+Kivy is a popular library for developing applications that can run across all major platforms.
```

Comment:
`developing applications`

Be specific. Applications can be anything -front end, backend. What does Kivy help to build?